styles.css

```diff
@@ -0,0 +1,265 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,700&display=swap');
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    list-style: none;
+}
+a {
+    text-decoration: none;
+}
+ul {
+    margin: 0;
+}
+body {
+    font-family: 'Noto Sans', sans-serif;
+    background: #f5f7fb;
+}
+header {
+    height: 54px;
+    overflow: hidden;
+    background: #1A294E;
+}
+header ul li a {
+    display: inline-block;
+    width: 100px;
+    text-align: center;
+    padding: 1rem 0.5rem;
+    color: #e0e0e0;
+}
+div ul, header ul {
+    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+    display: -ms-flexbox;      /* TWEENER - IE 10 */
+    display: -webkit-flex;
+    display: flex;
+    flex-direction: row;
```

Comment:
Vendor prefixes should go above the standard properties

Reply:
yes fixed